insta/insta_api/views.py

```diff
@@ -30,12 +30,60 @@ def post(self, request, *args, **kwargs):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+class PhotoDetailApiView(APIView):
```

Comment:
Every view should be in separate file. As I mentioned before let's split this structure per domain
```
insta/insta_api/views/photo_detail_view.py
...
```